### ISSUE TITLE
"Hello World" Webhook Example needed for Newcomers

### ISSUE THREAD

**Issue by timothydilbert** [KaotoIO/kaoto.io#23]
Most SaaS integrations are based on JSON webhooks. 

We need a "Hello World" example where a newcomer can create a webhook that accepts a JSON payload, extract a JSON property from the body, and write that property to a database. The example needs to include if/else conditions to show how to add basic business logic.

**Comment by lhein** [comment]
Good idea. Meanwhile you could check out https://github.com/KaotoIO/kaoto-examples. We have examples there for database access.
